src/mame/jaleco/ms1_tmap.cpp

```diff
@@ -18,6 +18,7 @@
 
 #include "emu.h"
 #include "ms1_tmap.h"
+#include "screen.h"
```

Comment:
If you’re including `"screen.h"` in the corresponding header, don’t do it here as well.